src/tests/functional/plugin/shared/src/behavior/ov_executable_network/get_metric.cpp

```diff
@@ -132,6 +132,15 @@ TEST_P(OVClassExecutableNetworkGetMetricTest_DEVICE_PRIORITY, GetMetricNoThrow)
     ASSERT_EQ(value, configuration[ov::device::priorities.name()].as<std::string>());
 }
 
+TEST_P(OVClassExecutableNetworkGetMetricTest_PERFORMANCE_HINT, GetMetricNoThrow) {
+    ov::Core ie = createCoreWithTemplate();
+    auto compiled_model = ie.compile_model(simpleNetwork, target_device, configuration);
+
+    ov::hint::PerformanceMode value;
+    OV_ASSERT_NO_THROW(value = compiled_model.get_property(ov::hint::performance_mode));
+    ASSERT_EQ(value, configuration[ov::hint::performance_mode.name()].as<ov::hint::PerformanceMode>());
+}
```

Comment:
can you check cases in SetPropertyComplieModelGetProperty, looks like some tests are overlapped?

Reply:
Updated, use SetPropertyComplieModelGetProperty to test perf_hint of AUTO and MULTI